### PR TITLE
[SA-1744] Added documentation for new metrics

### PR DESCRIPTION
**Pull request by comikal268**

### CODE REVIEW

File: content/api/metrics.apib [hunk 1]
```diff
@@ -49,7 +49,16 @@ Definitions for terms found in Metrics API
...
+| `count_inbox_panel`              | Panel messages delivered to the inbox.                                                              |
+| `count_spam_panel`               | Panel messages delivered to the spam folder.                                                        |
+| `count_inbox_seed`               | Seed messages delivered to the inbox.                                                               |
+| `count_spam_seed`                | Seed messages delivered to the spam folder.                                                         |
+| `count_moved_to_inbox`           | Panel messages delivered to the spam folder then moved to the inbox.                                |
+| `count_moved_to_spam`            | Panel messages delivered to the inbox then moved to the spam folder.                                |
 
+Note: The metrics `count_inbox_panel`,`count_spam_panel`,
+`count_inbox_seed`,`count_spam_seed`,`count_moved_to_inbox`,
+ and `count_moved_to_spam` are only available to the SparkPost Deliverability customers only.
```
Comment (gwalls) [inline content/api/metrics.apib:61]: Use the Deliverability tag, similar to the Enterprise tag

File: content/api/metrics.apib [hunk 2]
```diff
@@ -49,7 +49,16 @@ Definitions for terms found in Metrics API
 | `total_delivery_time_first`      | Total time taken to deliver messages on first attempt (milliseconds)                                |
 | `total_delivery_time_subsequent` | Total time taken to delivery messages on subsequent attempts (milliseconds)                         |
 | `count_unsubscribe`              | Total number of unsubscribes as a result of clicked links and the ISP list unsubscribe feature      |
+| `count_inbox_panel`              | Panel messages delivered to the inbox.                                                              |
```
Comment (gwalls) [inline content/api/metrics.apib:52]: Prefix the descriptions with the Deliverability tag (should link to the product page or wherever @haleysolomon thinks it should redirect to)

**Review by gwalls — CHANGES_REQUESTED** [review]
We should create a "Deliverability" tag similar to the "Enterprise" tag. We'll be able to use that in the new deliverability metrics descriptions (as well as the new routes and filter terms in the future).
image


I'm a little worried about overloading the term "deliverability" within our API docs. We already use that term in the numerous `metrics/deliverability/` routes. But that's ultimately product's call.

The new metric descriptions make sense to me, but @haleysolomon should probably sign off on them too.

**Review by gwalls — APPROVED** [review]
lgtm

**Review by mvanbosch — APPROVED** [review]
lgtm

**Pull request merged**